File: doc/.blinky/blinky.cs
using static System.Console;

WriteLine("\nMuntsOS C# LED Test\n");

// Configure a GPIO output to drive an LED

IO.Objects.SimpleIO.Device.Designator desg_LED =
  new IO.Objects.SimpleIO.Device.Designator(0, 26);

IO.Interfaces.GPIO.Pin LED =
  new IO.Objects.SimpleIO.GPIO.Pin(desg_LED,
    IO.Interfaces.GPIO.Direction.Output);

// Flash the LED forever (until killed)

WriteLine("Press CONTROL-C to exit.\n");

for (;;)
{
  LED.state = !LED.state;
  System.Threading.Thread.Sleep(500);
}

File: doc/.blinky/blinky.go
package main

import (
  "fmt"
  "time"
  "munts.com/interfaces/GPIO"
  "munts.com/objects/Error"
  "munts.com/objects/simpleio/Designator"
  "munts.com/objects/simpleio/GPIO_libsimpleio"
)

func main() {
  fmt.Printf("\nMuntsOS Go LED Test\n\n")

  // Configure a GPIO output to drive an LED

  var outp GPIO.Pin
  var err error

  outp, err = GPIO_libsimpleio.NewOutput(Designator.Designator { 0, 26 }, true)
  Error.Exit(err, "NewOutput() failed")

  var state bool

  // Flash the LED forever (until killed)

  fmt.Printf("Press CONTROL-C to exit.\n");

  for {
    state, err = outp.Get()
    Error.Exit(err, "outp.Get() failed")

    err = outp.Put(!state)
    Error.Exit(err, "outp.Put() failed")

    time.Sleep(500*time.Millisecond)
  }
}
